### DISCUSSION TITLE
Create a button to run all test suits

### DISCUSSION THREAD

**Discussion by TheProgrammer21** [Kong/insomnia#3879] [Ideas]
Create a button that runs all test suits from top to bottom.

**Comment by dimitropoulos** [comment]
I made a quick mockup:

![_snap_insomnia_150_resources_app asar_renderer html](https://user-images.githubusercontent.com/15232461/131534542-6f671a06-5ed7-4c5d-b7e8-72944d40314a.png)


The "Run All" button is the new part.
